app/packages/looker/src/elements/common/tags.ts

```diff
@@ -321,14 +321,12 @@ export class TagsElement<State extends BaseState> extends BaseElement<State> {
         if (field && LABEL_RENDERERS[field.embeddedDocType]) {
           if (path.startsWith("frames.")) continue;
           const classifications = LABEL_LISTS.includes(field.embeddedDocType);
-
-          if (!value.classifications?.length) continue;
-
           if (classifications) {
-            pushList(
-              LABEL_RENDERERS[field.embeddedDocType],
-              value.classifications
-            );
+            !value?.classifications?.length &&
+              pushList(
+                LABEL_RENDERERS[field.embeddedDocType],
+                value?.classifications
+              );
           } else {
             elements.push(LABEL_RENDERERS[field.embeddedDocType](path, value));
           }
```

Comment:
I put `if (!value.classifications?.length) continue;` inside `classifications` because classification bubbles are rendered in the `else` condition in L331. I am not sure if we actually need `if (!value.classifications?.length) continue;` 
Is `Classifications` type reserved for metadata?
@manivoxel51 @benjaminpkane

Reply:
not sure I understand the issue completelely. but `Classifications` is a valid label type.